### PR TITLE
Correct targetProduct in tiIndicator: submitTiIndicators

### PR DESCRIPTION
**Pull request by RecordedFutureOskbo**
Required items, please complete
   
   Change(s):
   - Change "targetProduct": "Microsoft Sentinel" back to "Azure Sentinel"
   - Generated new solution 2.2.2.zip containing the fix.

   Reason for Change(s):
Error in our playbooks introduced due to name change from Azure Sentinel to Microsoft Sentinel. Introduced in this [COMMIT](https://github.com/Azure/Azure-Sentinel/commit/4275adc96a96cfaabaddb3406aed7360bca1d195).

This fix will correct the targetProduct back to Azure Sentinel according to the spec https://learn.microsoft.com/en-us/graph/api/tiindicator-submittiindicators?view=graph-rest-beta&tabs=http

   Testing Completed:
   - Installed mainTemplate.json
   - Installed playbooks from templates
   - Run templates importing data and triggered analytic rules.

### CODE REVIEW

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo, please check if there is need to change the text from "Microsoft Sentinel" to "Azure sentinel" again? as we have changed at all instances, thanks.

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo Ok, let me check if we can skip these failing validations, thanks.

**Comment by RecordedFutureOskbo** [comment]
@v-sabiraj any status on this? I will try to create a workaround so we can get this fix to our customers.

**Comment by RecordedFutureOskbo** [comment]
@v-sabiraj, @v-atulyadav any new on this PR? I have fixed all Required checks.

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo, actually the arm-ttk for particular playbooks is failing, the check has been added to check individual files.

File: Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
                                             "domainName": "@{body('Parse_JSON')?['Name']}",
                                             "expirationDateTime": "@{addHours(utcNow(),2)}",
                                             "ingestedDateTime": "@{utcNow()}",
-                                            "targetProduct": "Microsoft Sentinel",
+                                            "targetProduct": "@{concat('Azure ', 'Sentinel')}",
```
Comment (v-sabiraj) [inline Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json:128]: @RecordedFutureOskbo, there's no need to add concat we skipping the check for targetProduct, you can revert it to  'Azure Sentinel', thanks.

**Comment by RecordedFutureOskbo** [comment]
> @RecordedFutureOskbo, actually the arm-ttk for particular playbooks is failing, the check has been added to check individual files.

I have a separate branch where I tried to remove all arm-ttk issues. I rather submit it separate, since I will rename all playbooks to azuredeploy.json as one of the updates.

**Comment by RecordedFutureOskbo** [comment]
@v-sabiraj do you need fix for arm-ttk in this PR?

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo, this is fixed, will approve, thanks.

**Review by v-sabiraj — APPROVED** [review]

**Comment by v-sabiraj** [comment]
Hey, @RecordedFutureOskbo can you please update your branch from master. The fix has been done in master, thanks.

**Comment by RecordedFutureOskbo** [comment]
> Hey, @RecordedFutureOskbo can you please update your branch from master. The fix has been done in master, thanks.

Done, thanks!

**Pull request merged**